Packs/Base/Scripts/CreateIndicatorRelationship/CreateIndicatorRelationship.py

```diff
@@ -0,0 +1,196 @@
+
+    :return: raise an error if one of the validations fail.
+    :rtype: ``None``
+    """
+    if len(argToList(args.get('entity_a'))) > 1:
+        raise Exception("entity_a is a list, Please insert a single entity_a to create the relationship")
+    if len(argToList(args.get('entity_b_type'))) > 1:
+        raise Exception("entity_b_type is a list, Please insert a single type to create the relationship")
+    if args.get('entity_b') and args.get('entity_b_type') and args.get('entity_b_query'):
+        raise Exception("entity_b_query can not be used with entity_b and/or entity_b_type")
+    if not args.get('entity_b_query') and not args.get('entity_b'):
+        raise Exception("Missing entity_b in the create relationships")
+    if args.get('entity_b') and not args.get('entity_b_type'):
+        raise Exception("Missing entity_b_type in the create relationships")
+    return args
+
+
+def create_indicators(args: dict):
+    """
+    When the create_indicator argument is set to True, create the new indicator first.
+
+    :type args: ``dict``
+    :param args: dict of arguments of the command.
+
+    :return: return a list of errors, empty if no errors.
+    :rtype: ``None``
+    """
+    entity_b_to_create = remove_existing_entity_b_indicators(args)
+    indicators = []
+    for entity_b in entity_b_to_create:
+        indicator = {
+            'value': entity_b,
+            'type': args.get('entity_b_type'),
+        }
+        indicators.append(indicator)
+    errors = list()
+    for indicator in indicators:
+        res = demisto.executeCommand("createNewIndicator", indicator)
+        if is_error(res[0]):
+            errors.append("Error creating indicator - {}".format(res[0]["Contents"]))
+    demisto.debug(
+        f"Create Indicators From CreateIndicatorRelationship automation: {len(indicators) - len(errors)}"
+        f" indicators were created with values {str(indicators)}"
+    )
+    if errors:
+        demisto.debug(f'Errors were found while creating indicators from CreateIndicatorRelationships automation:'
+                      f' {json.dumps(errors, indent=4)}')
+
+
+def create_relationships(args: dict) -> Tuple[List[EntityRelation], str]:
+    """
+    Create relationships from given arguments.
+
+    :type args: ``dict``
+    :param args: dict of arguments of the command.
+
+    :return: relationships object with human readable output.
+    :rtype: ``tuple``
+    """
+    if args.get('entity_b_query'):
+        relationships = create_relation_command_using_query(args)
+    else:
+        relationships = create_relationships_with_args(args)
+
+    if len(relationships) == 1:
+        human_readable = f"Relationship for {args.get('entity_a')} was created successfully."
+    elif len(relationships) > 1:
+        human_readable = f"Relationships for {args.get('entity_a')} were created successfully."
+    else:
+        human_readable = f"Relationships were not created for {args.get('entity_a')}."
```

Comment:
please add the exact number of relationships created in the title.

Reply:
done